centreon/cron/centreon-backup.pl

```diff
@@ -325,15 +326,26 @@ ()
 }
 
 sub getMySQLConfFile() {
+    my @cnf_files = (
+        # EL
+        '/etc/my.cnf.d/centreon.cnf',
+        # Debian
+        '/etc/mysql/conf.d/80-centreon.cnf',
+        '/etc/mysql/mariadb.conf.d/80-centreon.cnf',
+        '/etc/mysql/mysql.conf.d/80-centreon.cnf',
+        # fallback
+        '/etc/my.cnf');
     if (defined($MYSQL_CONF)) {
-        if ( -e $MYSQL_CONF) {
-            return $MYSQL_CONF;
-        } elsif ( -e '/etc/my.cnf' ) {
-            return '/etc/my.cnf';
-        }    
-    } else {
-        print STDERR "Unable to get Mysql configuration\n";
+        unshift @cnf_files, $MYSQL_CONF;
+    }
```

Comment:
optionnal : you could have used the if to check file existance here instead of adding to the array
```suggestion
    if (-e $MYSQL_CONF) {
        return $MYSQL_CONF;
    }
```